content/en/user-guide/aws/lakeformation/index.md

```diff
@@ -0,0 +1,101 @@
+
+LocalStack allows you to use the Lake Formation APIs in your local environment to register resources, grant permissions, and list resources and permissions. The supported APIs are available on our [API coverage page](https://docs.localstack.cloud/references/coverage/coverage_lakeformation/), which provides information on the extent of Lake Formation's integration with LocalStack.
+
+## Getting started
+
+This guide is designed for users new to Lake Formation and assumes basic knowledge of the AWS CLI and our [`awslocal`](https://github.com/localstack/awscli-local) wrapper script.
+
+Start your LocalStack container using your preferred method. We will demonstrate how to register an S3 bucket as a resource in Lake Formation, grant permissions to a user, and list the resources and permissions.
+
+### Register the resource
+
+Create a new S3 bucket named `lf-emr-athena-result-123` using the `mb` command:
+
+{{< command >}}
+$ awslocal s3 mb s3://lf-emr-athena-result-123
+{{</ command >}}
+
+You can now register the S3 bucket as a resource in Lake Formation using the [`RegisterResource`](https://docs.aws.amazon.com/lake-formation/latest/dg/API_RegisterResource.html) API. Create a file named `input.json` with the following content:
+
+```json
+{
+    "ResourceArn": "arn:aws:s3:::lf-emr-athena-result-123",
+    "UseServiceLinkedRole": true
+}
+```
+
+Run the following command to register the resource:
+
+{{< command >}}
+awslocal lakeformation register-resource \
+    --cli-input-json file://input.json
+{{</ command >}}
+
+### List resources
+
+You can list the registered resources using the [`ListResources`](https://docs.aws.amazon.com/lake-formation/latest/dg/API_ListResources.html) API. Execute the following command to list the resources:
+
+{{< command >}}
+awslocal lakeformation list-resources
+{{</ command >}}
+
+The following output is displayed:
+
+```bash
+{
+    "ResourceInfoList": [
+        {
+            "ResourceArn": "arn:aws:s3:::lf-emr-athena-result-123",
+            "LastModified": "2024-07-11T23:27:30.699312+05:30"
+        }
+    ]
+}
+```
+
+### Grant permissions
+
+You can grant permissions to a user or group using the [`GrantPermissions`](https://docs.aws.amazon.com/lake-formation/latest/dg/API_GrantPermissions.html) API. Create a file named `permissions.json` with the following content:
+
+```json
+{
+    "CatalogId": "000000000000",
+    "Principal": {
+        "DataLakePrincipalIdentifier": "arn:aws:iam::000000000000:user/lf-developer"
+    },
+    "Resource": {
+        "Table": {
+            "CatalogId": "000000000000",
+            "DatabaseName": "tpc",
+            "TableWildcard": {}
+        }
+    },
+    "Permissions": [
+        "SELECT"
+    ],
+    "PermissionsWithGrantOption": []
+}
+```
+
+Run the following command to grant permissions:
+
+{{< command >}}
+$ awslocal lakeformation grant-permissions \
+    --cli-input-json file://check.json
+{{</ command >}}
+
```

Comment:
I don't know if that is the case here as well, but without setting `ENFORCE_IAM`, setting permissions is just "cosmetic". So if that is the case here as well, we should maybe hint at that and at the environment variable here.

Reply:
Yeah, the user guides currently aren't at par with AWS since we skip mentioning the IAM enforcement feature or setting the right permissions. Will revisit this later.